src/components/drag-and-drop/with-drag/with-drag.js

```diff
@@ -53,7 +53,7 @@ class WithDrag extends React.Component {
   // In Safari it changes the mouse cursor when dragging because it thinks text is being selected
   // We test if the target is an html element (not text) or if we already know the user is dragging
   allowTextSelection = (event) => {
-    if ((event.target instanceof HTMLElement) || this.dragging) {
+    if (event.target.tagName === 'SPAN' || this.dragging) {
```

Comment:
this is true for `Icon`, but this component is not limited to just dragging `Icon` components

Reply:
@andrewjtait What other elements should we protect against?

Reply:
what about checking if the tag is a HTMLElement but not an Input or something?

Reply:
@andrewjtait I think it would work, we will see how to refactor that. 👍